src/components/markdown-renderer/replace-components/link-replacer/link-replacer.tsx

```diff
@@ -25,7 +25,13 @@ export class LinkReplacer extends ComponentReplacer {
       return undefined
     }
 
-    const url = node.attribs.href
+    const url = node.attribs.href.trim()
+
+    // eslint-disable-next-line no-script-url
+    if (url.startsWith('data:') || url.startsWith('javascript:')) {
+      return <span>{ node.attribs.href }</span>
```

Comment:
Maybe we want to use the trimmed url here?

Reply:
I'm unsure. I just thought to output the user input as unmodified as possible.

What do you think, @mrdrogdrog?

Reply:
TBH I don't care here. The trim is necessary to make sure that you can't bypass our check with spaces.